spark/src/main/java/org/opensearch/sql/spark/dispatcher/BatchQueryHandler.java

```diff
@@ -57,9 +57,9 @@ protected JSONObject getResponseFromExecutor(AsyncQueryJobMetadata asyncQueryJob
 
   @Override
   public String cancelJob(AsyncQueryJobMetadata asyncQueryJobMetadata) {
```

Comment:
This handles StreamingQueryHandler as well right.

Reply:
refactor code, support cancel refresh statement in latest commit.